pkg/ops/traceflow.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ops
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/ops/v1alpha1"
+)
+
+// NewTraceflow fills up Spec field in a Traceflow.
+func NewTraceflow(src v1alpha1.Source, dst v1alpha1.Destination, pkt v1alpha1.Packet) *v1alpha1.Traceflow {
+	var name string
+	if len(dst.Pod) != 0 {
+		name = src.Namespace + "-" + src.Pod + "-to-" + dst.Namespace + "-" + dst.Pod
+	} else if len(dst.Service) != 0 {
+		name = src.Namespace + "-" + src.Pod + "-to-" + dst.Namespace + "-" + dst.Service
+	} else {
+		name = src.Namespace + "-" + src.Pod + "-to-" + dst.IP
+	}
```

Comment:
should we keep a random suffix in the name? In case some test wants to generate different traceflow requests with the same source and destination.

Reply:
Good idea. I refactored code about generating a traceflow. Now this file is deleted, all traceflow config is in traceflow_test.go.